gitlint-core/gitlint/tests/contrib/rules/test_disallow_fixup_squash_commit.py

```diff
@@ -0,0 +1,34 @@
+
+# -*- coding: utf-8 -*-
+from gitlint.tests.base import BaseTestCase
+from gitlint.rules import RuleViolation
+from gitlint.contrib.rules.signedoff_by import SignedOffBy
```

Comment:
Importing the wrong class here, hence why CI is failing
```suggestion
from gitlint.contrib.rules.signedoff_by import DisallowCleanupCommits
```

Note the change of classname to `DisallowCleanupCommits` as discussed, this should be updated in other places too.

Reply:
Meh, always missing things like this. Thanks for taking the time!